### PR TITLE
Add governance proposal to transfer out Morpho tokens

### PR DESCRIPTION
**Pull request by shahthepro**
## Governance proposal
- Proposal ID: [65964300454407724775503884779095378039055881377810492835401823493288595929298](<https://originprotocol.eth.limo/#/more/1:0x1d3fbd4d129ddd2372ea85c5fa00b2682081c9ec:65964300454407724775503884779095378039055881377810492835401823493288595929298>)
- Proposal Tx: [0x290ce4205d0bc6df0220f743b106a41290297c94bf914400db646d496ceeb5b5](<https://etherscan.io/tx/0x290ce4205d0bc6df0220f743b106a41290297c94bf914400db646d496ceeb5b5>)

## Deploy checklist

Two reviewers complete the following checklist:

```
- [ ] All deployed contracts are listed in the deploy PR's description
- [ ] Deployed contract's verified code (and all dependencies) match the code in master
- [ ] Contract constructors have correct arguments
- [ ] The transactions that interacted with the newly deployed contract match the deploy script.
- [ ] Governance proposal matches the deploy script
- [ ] Smoke tests pass after fork test execution of the governance proposal
```

### CODE REVIEW

**Review by naddison36 — APPROVED** [review]
LGTM

File: contracts/deploy/mainnet/109_transfer_morpho.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    );
+    const cOUSDMorphoCompoundProxy = await ethers.getContract(
+      "MorphoCompoundStrategyProxy"
+    );
+    const cOETHMorphoAaveProxy = await ethers.getContract(
+      "OETHMorphoAaveStrategyProxy"
+    );
+
+    const cOUSDMorphoAave = await ethers.getContractAt(
+      "InitializableAbstractStrategy",
```
Comment (sparrowDom) [inline contracts/deploy/mainnet/109_transfer_morpho.js:25]: shouldn't this be `MorphoAaveStrategy`

Comment (sparrowDom) [reply contracts/deploy/mainnet/109_transfer_morpho.js:25]: the same for below 2 strategies

Comment (sparrowDom) [reply contracts/deploy/mainnet/109_transfer_morpho.js:25]: Ah nvm I see the function is in `InitializableAbstractStrategy`

**Review by sparrowDom — APPROVED** [review]
LGTM

**Pull request merged**